CHANGELOG.md

```diff
@@ -16,12 +16,14 @@
 - Remove TypeMeta declaration from the implementation of the objects [#1462](https://github.com/operator-framework/operator-sdk/pull/1462/)
 - Relaxed API version format check when parsing `pkg/apis` in code generators. API dir structures can now be of the format `pkg/apis/<group>/<anything>`, where `<anything>` was previously required to be in the Kubernetes version format, ex. `v1alpha1`. ([#1525](https://github.com/operator-framework/operator-sdk/pull/1525))
 - The SDK and operator projects will work outside of `$GOPATH/src` when using [Go modules](https://github.com/golang/go/wiki/Modules). ([#1475](https://github.com/operator-framework/operator-sdk/pull/1475))
+-  `CreateMetricsService()` function from the metrics package accepts an array of ServicePort objects ([]v1.ServicePort) as input to create Service metrics. `CRPortName` constant is added to describe the string of custom resource port name. ([#1560](https://github.com/operator-framework/operator-sdk/pull/1560))
 
 ### Deprecated
 
 ### Removed
 
 - The SDK no longer depends on a `vendor/` directory to manage dependencies *only if* using [Go modules](https://github.com/golang/go/wiki/Modules). The SDK and operator projects will only use vendoring if using `dep`, or modules and a `vendor/` dir is present. ([#1519](https://github.com/operator-framework/operator-sdk/pull/1519))
+- `ExposeMetricsPort` is now depracted and was replaced with `CreateMetricsService()` function. `PrometheusPortName` constant is replaced with `OperatorPortName`. ([#1560](https://github.com/operator-framework/operator-sdk/pull/1560))
```

Comment:
Deprecated means that the thing still exists and will be removed in a future version, right? In this PR, we're actually removing it without deprecating it.

So that leads me to a question: should we actually deprecated `ExposeMetricsPort` and leave it in place for now?

Reply:
we are pre `1.0` I think it is ok to deprecate and replace in a single release IMO. WDYT?

Reply:
I think we said we are making breaking changes until the 1.0 release yes.

Reply:
Maybe we could make it more in bold in the CHANGELOG? Also with the migration doc coming up that you mention @joelanford it should be easy for people to migrate IMO.

Reply:
Yeah, I think it's fine to remove it. Just wanted to make sure we were all in agreement to do that. I would suggest:
- Prefixing with `**Breaking change**:` like we've done before
- `s/deprecated/removed/` to make it clear the `ExposeMetricsPort` is no longer present.

WDYT?

Reply:
sgtm, done!